index.html

```diff
@@ -1,12 +1,143 @@
+            </button>
+            <div class="content">
+              <img
+                class="card-img"
+                src="https://media.proprofs.com/images/QM/user_images/2503852/New%20Project%20(83)(90).jpg"
+                alt=" the office "
+              />
+
+              <div class="card-header">
+                <h2>The Office Quiz for true fans</h2>
+
+                <div class="card-body">
+                  <p>Take this quiz to test yourself</p>
+                  <p>20 Questions</p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <!-- card start -->
+          <div class="card w-30 basic">
+            <button class="icon favourite-icon">
+              <i class="fa-regular fa-heart fa-2x"></i>
+            </button>
+            <div class="content">
+              <img
+                class="card-img"
+                src="https://media.proprofs.com/images/QM/user_images/2503852/New%20Project%20(83)(90).jpg"
+                alt=" the office "
+              />
+
+              <div class="card-header">
+                <h2>The Office Quiz for true fans</h2>
+
+                <div class="card-body">
+                  <p>Take this quiz to test yourself</p>
+                  <p>20 Questions</p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <!-- card start -->
+          <div class="card w-30 basic">
+            <button class="icon favourite-icon">
+              <i class="fa-regular fa-heart fa-2x"></i>
+            </button>
+            <div class="content">
+              <img
+                class="card-img"
+                src="https://media.proprofs.com/images/QM/user_images/2503852/New%20Project%20(83)(90).jpg"
+                alt=" the office "
+              />
+
+              <div class="card-header">
+                <h2>The Office Quiz for true fans</h2>
+
+                <div class="card-body">
+                  <p>Take this quiz to test yourself</p>
+                  <p>20 Questions</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="section flex-col leaderboard-sec">
+          <h1 class="section-heading">Leaderboard</h1>
+            <div class="leader-list">
+                <div class="container leader-sec flex-align-center">
+                    <div class="leader-rank">#1</div>
+                    <div class="leader-img">
+                        <img alt="leader avatar" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQGtnn5CQr_nEzmp2EDlJAvgXaA4n1taH-sDQ&usqp=CAU">
+                    </div>
+                    <div class="flex-col leader-name">
+                        <div><strong>Avenger</strong></div>
+                        <div>Bhavika Tibrewal</div>
+                    </div>
+                    <div class="body-l leader-points primary">300pts</div>
+                </div>
```

Comment:
300pts is a text, so you can replace div with a p tag which i think would be more accesible

Reply:
Definately that should have been a p tag, will change